code/game/objects/items/holy_weapons.dm

```diff
@@ -498,6 +500,38 @@
 		to_chat(user, "The blade is dormant. Maybe you can try again later.")
 		possessed = FALSE
 
+/obj/effect/proc_holder/possessed_throw
+	name = "Launch Item"
+	desc = "Gather your energy to throw your possessed item."
+	var/cooldown_time = 30 // Time, in deciseconds, between throws
+	var/cooldown = 0
+
+/obj/effect/proc_holder/possessed_throw/Click()
+	var/message
+	if(active)
+		message = "<span class='notice'>You disperse your energy.</span>"
+		remove_ranged_ability(message)
+	else if(cooldown <= world.time)
+		message = "<span class='notice'>You gather your energy. <B>Left-click to throw yourself!</B></span>"
+		add_ranged_ability(usr, message, TRUE)
+	else
+		to_chat(usr, "You are still building up your energy!")
+
+/obj/effect/proc_holder/possessed_throw/InterceptClickOn(mob/living/caller, params, atom/target)
+	. = ..()
+	if(istype(caller.loc, /obj/item)) // If the shade is possessing something
+		var/obj/item/I = caller.loc
+		if(istype(I.loc, /mob/living)) // If embedded or being held
+			caller.visible_message("[src] rips free of [I.loc]!", "You rip free of [I.loc]!")
+			I.forceMove(get_turf(I.loc)) // Unembed, or drop
```

Comment:
Does this actually work and not break embed somehow?

Reply:
It's been removed now, and this is how the embed system un-embeds already